tests/test_http_parser.py

```diff
@@ -1761,6 +1761,69 @@ async def test_parse_chunked_payload_split_end_trailers4(
         assert out.is_eof()
         assert b"asdf" == b"".join(out._buffer)
 
+    async def test_parse_chunked_payload_split_CRLF(
```

Comment:
Can we get a test that uses the `response: HttpResponseParser` fixture, rather than using HttpPayloadParser directly?
I think we'd want to ensure that both parsers work the same way, this only tests the Python parser (not used by most users).

Reply:
Which is this second parser ?
Please bear with me, I have only been looking at aiohttp source code for few days.

The tests check the changes made in HttpPayloadParser, not in the second parser.

Reply:
Yes, if you use the fixture as mentioned above, the test will be run against both parsers.

Reply:
You did not answer my question. Please, what is the name of the second parser ?

Why should changes that affect HttpPayloadParser be tested against the second parser ?

What if the tests against the second parser fail ?
In that case:
- do you consider that these failures must be fixed within this PR ? 
- would'nt it be better to update the title of the PR and the issue to restrict their scope to HttpPayloadParser ?

Reply:
There's a Python parser and a C parser (llhttp). Both should behave the same way, hence why we have a fixture to run the same tests against both parsers.